samigo/samigo-app/src/webapp/jsf/delivery/item/deliverCalculatedQuestion.jsp

```diff
@@ -27,10 +27,10 @@ should be included in file importing DeliveryMessages
   <!-- ATTACHMENTS -->
 <%@ include file="/jsf/delivery/item/attachment.jsp" %>
 
-<samigo:dataLine value="#{question.finArray}" var="answer"
-  separator=" " first="0" rows="100">
+<samigo:dataLine value="#{question.finArray}" var="answer" separator=" " first="0" rows="100">
   <h:column>
-      <h:outputText value="#{answer.text} " escape="false" />
+      <h:outputLabel for="calcq" value="#{answer.text} " escape="false" />
+      <f:verbatim>&nbsp;</f:verbatim>
```

Comment:
Sure, ideally. But I'm looking to make a number of a11y changes quickly without spending a ton of time on each of these.